config/systems.yaml

```diff
@@ -4,10 +4,10 @@ systems:
   aliases: [kind-kind]
   overlays: [kind, overlays/kind]
   master: kind-control-plane
-  workers:
+  workers: [kind-worker]
   rabbits:
-    kind-worker: {0: compute-01, 1: compute-02, 6: compute-03}
-    kind-worker2: {4: compute-04}
+    kind-worker2: {0: compute-01, 1: compute-02, 6: compute-03}
+    kind-worker3: {4: compute-04}
```

Comment:
`kind-worker` is now the control plane, right?

Reply:
It's a worker, neither the control plane nor a tainted nnf-node.

Reply:
It's the non-rabbit worker node.  SLCs run here, so that's things like dws-operator and the deployment (versus the daemonset) from nnf-sos.